scripts/sct_utils.py

```diff
@@ -206,6 +206,99 @@ def run(cmd, verbose=1, error_exit='error', raise_exception=False):
         return status_output, output_final[0:-1]
 
 
+def check_exe(name):
+    """
+    Ensure that a program exists
+    :type name: string
+    :param name: name or path to program
+    :return: path of the program or None
+    """
+    def is_exe(fpath):
+        return os.path.isfile(fpath) and os.access(fpath, os.X_OK)
+
+    fpath, fname = os.path.split(name)
+    if fpath and is_exe(name):
+        return fpath
+    else:
+        for path in os.environ["PATH"].split(os.pathsep):
+            path = path.strip('"')
+            exe_file = os.path.join(path, name)
+            if is_exe(exe_file):
+                return exe_file
+
+    return None
+
+
+def display_viewer(l_file, l_colormap=[], l_minmax=[], l_opacity=[], mode='', verbose=1):
```

Comment:
Hungarian notation is used only once in the rest of the code, so `l_` feels a bit weird, but hey, it works.

Reply:
Uh no, it's used everywhere but with `list_`. my bad.

Reply:
yeah... another thing we can discuss wrt. convention. I started with `list_` but found it overloaded the code...

Reply:
In Python almost nobody is using type names (or prefixes) in variable names, it almost always overloads the code. Using plural in the variable name is usually good enough to make the reader understand that a list-like object is on hand.

Reply:
fixed in b88fa9bba23e9428d8316c4616f743c70a8bd4f8